apps/experiments/migrations/preload_data/aws_voices.json

```diff
@@ -5,651 +5,744 @@
         "language_code": "arb",
         "name": "Zeina",
         "gender": "Female",
-        "neural": false
+        "neural": false,
+        "service": "AWS"
       },
       {
         "language": "Arabic (Gulf)",
         "language_code": "ar-AE",
         "name": "Hala",
         "gender": "Female",
-        "neural": true
+        "neural": true,
+        "service": "AWS"
```

Comment:
Nit: Technically we don't have to add the "service" field to each entry, since we know this file contains onyl AWS voices. We can add the service field when loading them

Reply:
Good point. One advantage of keeping it is that it stays with the object and/or we could merge the list of objects together in the future. It does use more characters, though. (I'm going to leave it unless you feel strongly)

Reply:
No strong feelings about this. Good to leave it as is